lib/fake_artist/table.ex

```diff
@@ -10,18 +10,43 @@ defmodule FakeArtist.Table do
     GenServer.call(pid, {:update_name_tag, {id, name_tag}})
   end
 
+  def add_player(table_pid, player_pid) do
+    GenServer.call(table_pid, {:add_player, player_pid})
+  end
+
   # Server Callbacks
   def init([name]) do
-    {:ok, {%{}, name}}
+    {:ok, {%{}, name, 0}}
   end
 
-  def handle_call({:update_name_tag, {id, name_tag}}, _from, {state_map, table_name}) do
-    state_map = Map.put(state_map, id, name_tag)
+  def handle_call({:update_name_tag, {id, name_tag}}, _from, {id_map, table_name, user_count}) do
+    id_map = Map.put(id_map, id, name_tag)
 
     FakeArtistWeb.Endpoint.broadcast("table:#{table_name}", "update", %{
-      names: Map.values(state_map)
+      names: Map.values(id_map)
     })
 
-    {:reply, :ok, {state_map, table_name}}
+    {:reply, :ok, {id_map, table_name, user_count}}
+  end
+
+  def handle_call({:add_player, player_pid}, _from, {id_map, table_name, user_count}) do
+    IO.puts("started monitoring")
+    IO.inspect(player_pid)
```

Comment:
btw, you can combine the previous 2 lines you just did in IO.puts/Logger calls like this:
```elixir
IO.puts("started monitoring #{inspect(player_pid)}"